.github/workflows/test.yaml

```
@@ -16,9 +16,12 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
     steps:
-      - uses: ros-tooling/setup-ros@0.0.23
-      - uses: ros-tooling/action-ros-ci@0.0.17
+      - uses: ros-tooling/setup-ros@v0.1
         with:
+          required-ros-distributions: foxy
+      - uses: ros-tooling/action-ros-ci@v0.1
+        with:
+          target-ros2-distro: foxy
```

This definitely isn't clear enough in the docs, and the default setting right now is probably the wrong experience.
If you want to build only against installed binary dependencies instead of checking out `ros2.repos`, you can specify an empty `vcs-repo-file-url` input parameter, which will override the default value of `https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos`

```suggestion
        with:
          target-ros2-distro: foxy
          vcs-repo-file-url: ""
```

---

perfect! That seems to work. Thanks for the help.
And yes, that would be great to document, because I feel that's a valid use case for this GH action.